src/main/java/com/google/gcp_variant_transforms/library/VariantToBqUtils.java

```diff
@@ -0,0 +1,241 @@
+// Copyright 2020 Google LLC
+
+package com.google.gcp_variant_transforms.library;
+
+import com.google.api.services.bigquery.model.TableRow;
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.Genotype;
+import htsjdk.variant.variantcontext.GenotypesContext;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFHeader;
+import htsjdk.variant.vcf.VCFHeaderLineCount;
+import htsjdk.variant.vcf.VCFHeaderLineType;
+import htsjdk.variant.vcf.VCFInfoHeaderLine;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Utility constants and methods for write values with defined value type into Big Query rows.
+ */
+public class VariantToBqUtils {
+
+  private static final String MISSING_FIELD_VALUE = ".";
+  private static final String PHASESET_FORMAT_KEY = "PS";
+  private static final String DEFAULT_PHASESET = "*";
+  private static final int MISSING_GENOTYPE_VALUE = -1;
+
+  /**
+   * Constants for column names in the BigQuery schema.
+   */
+  public static class ColumnKeyConstants {
```

Comment:
Generating the schema uses these ColumnKeyConstants. Maybe we could rename this class to indicate the utilities here are used in multiple places (not just VariantToBq); alternatively, we could take out just the static class ColumnKeyConstants and put it in a shared utility class for VariantToBq and SchemaGeneration

Reply:
Agree. Have you add a Constants class in your PR? If so I will follow the code you have written and make change in my PR

Reply:
I have created a common/Constants.java to store all constants and field names